nextcord/flags.py

```diff
@@ -486,6 +487,79 @@ def all(self) -> List[UserFlags]:
         return [public_flag for public_flag in UserFlags if self._has_flag(public_flag.value)]
 
 
+@fill_with_flags()
+class MemberFlags(BaseFlags):
+    r"""Wraps up the Discord Guild Member flags
+
+    .. versionadded:: 2.5
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two MemberFlags are equal.
+
+        .. describe:: x != y
+
+            Checks if two MemberFlags are not equal.
+
+        .. describe:: x | y, x |= y
```

Comment:
`__xor__`, `__invert__`, `__and__`, `__or__` and the relevant in-place variants currently are not implemented for flag values.